tests/System.Text.Primitives.Tests/PrimitiveParserUInt32PerfTests.cs

```diff
@@ -279,23 +279,28 @@ private unsafe static void PrimitiveParserByteSpanToUInt32(string text)
             }
         }
 
-        [Benchmark(Skip = "The generic type 'System.Collections.Generic.List`1' was used with an invalid instantiation in assembly 'System.Private.CoreLib")]
-        private unsafe static void PrimitiveParserByteSpanToUInt32_VariableLength()
+        [Benchmark]
+        [InlineData(0)]
```

Comment:
I suppose. I don't really understand why we have two. This isn't testing for correctness so I don't see the significance of modifying the length of the string being parsed during each inner loop.